sql/core/src/test/resources/sql-tests/results/csv-functions.sql.out

```diff
@@ -43,7 +43,7 @@ struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
 {
-  "errorClass" : "_LEGACY_ERROR_TEMP_1227",
+  "errorClass" : "INVALID_DATA_TYPE_SCHEMA",
   "messageParameters" : {
     "e1" : "\n[PARSE_SYNTAX_ERROR] Syntax error at or near 'InvalidType': extra input 'InvalidType'(line 1, pos 2)\n\n== SQL ==\na InvalidType\n--^^^\n",
```

Comment:
How about to unwrap this error? When both attempts fail, just bypass the first one to users.